controls/3_5_uncommon_network_protocols.rb

```diff
@@ -17,17 +17,17 @@
 
 title '3.5 Uncommon Network Protocols'
 
-control 'cis-dil-benchmark-3.5.1' do
+control ' ' do
```

Comment:
We should not remove a control id, any specific reason why you removed this?

Reply:
Hi,

It was a mistake from my end. Thanks for noticing I will fix it ASAP.